subsys/bluetooth/shell/bt.c

```diff
@@ -2032,6 +2035,16 @@ static bool adv_param_parse(size_t argc, char *argv[],
 
 			param->peer = &addr;
 			argn += 2;
+		} else if (!strcmp(arg, "interval")) {
```

Comment:
What's the rationale behind just adding a `interval` rather than an `interval-min` and `interval-max`?

Reply:
Note that `interval` requires two values (min and max).

The reasoning is that if you want to specify a non-default range for the interval, you would always want to specify both the minimum and the maximum. What would the other value be if you only specify one of them?